recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
-{% set version = "2.2.27.27" %}
+{% set version = "2.2.28" %}
 
 package:
   name: udunits2
   version: {{ version }}
 
 source:
-  url: https://github.com/Unidata/UDUNITS-2/archive/v{{ version }}.tar.gz
-  sha256: 6c65b3097ef38b94c01dfede8904346c1439de9704b9bae7deea72bd5143a7bb
+  url: https://artifacts.unidata.ucar.edu/repository/downloads-udunits/{{ version }}/udunits-{{ version }}.tar.gz
```

Comment:
@conda-forge/udunits2, it seems that the 2.2.28 tag was never created on GitHub.  Thus the change in location for the source.  We can change back if tagging happens as expected for future releases.  See:
https://github.com/Unidata/UDUNITS-2/issues/104
and
https://github.com/Unidata/UDUNITS-2/issues/109

Reply:
I would consider the new location to be the canonical location.

Reply:
Hopefully, the bot will understand it well enough to notice new releases.